tests/test_global.py

```diff
@@ -9,33 +9,127 @@
 from satellite_images_nso_tif_model_iterator.filenames.file_name_generator import (
     OutputFileNameGenerator,
 )
+from satellite_images_nso_tif_model_iterator.h3_hexagons.nso_tif_model_iterater_hexagon_output import (
+    output_h3_hexagons_from_pixels,
+)
+from pathlib import Path
 import pandas as pd
+import geopandas as gpd
+
 
 # import settings_test
 
+
 # All these variables are local!
 # TODO: pytest does not seem to find the settings_test file after several attempts, for now hardcoded settings in the file.
-model_path = "C:/repos/satellite-images-nso-datascience/saved_models/Superview_Nieuwkoopse_plassen_20190302_113613_to_20221012_104900_random_forest_classifier.sav"
-test_tif_files_dir = "E:/output/test/Nieuwkoopse_plassen/*SV*.tif"
-output_path_test = "E:/output/test/Nieuwkoopse_plassen/"
-output_path = "E:/output/test/Nieuwkoopse_plassen/"
+model_path_sv = "C:/repos/satellite-images-nso-datascience/saved_models/Superview_Nieuwkoopse_plassen_20190302_113613_to_20221012_104900_random_forest_classifier.sav"
+test_tif_files_dir_sv = "E:/output/test/Nieuwkoopse_plassen/input_data/*SV*.tif"
+
+
+model_path_pneo = "C:/repos/satellite-images-nso-datascience/saved_models/PNEO_Nieuwkoopse_plassen_20230603_to_20230905_random_forest_classifier.sav"
+test_tif_files_dir_pneo = "E:/output/test/Nieuwkoopse_plassen/input_data/*PNEO*.tif"
+
+
+a_tif_file_hexagon_test = "E:/output/test/Nieuwkoopse_plassen/input_data/20230905_105231_PNEO-03_1_1_30cm_RD_12bit_RGBNED_Uithoorn_Nieuwkoopse_Plassen_De_Haeck_cropped_ndwi_re_ndvi_Ground_test.tif"
+
+output_path_test = "E:/output/test/Nieuwkoopse_plassen/output/"
+
+# Functions
+
 
-final_artefact = pickle.load(open(model_path, "rb"))
+def is_valid_geodataframe_with_polygons(df):
```

Comment:
You also have this functionality: 
https://geopandas.org/en/stable/docs/reference/api/geopandas.testing.assert_geodataframe_equal.html